### DISCUSSION TITLE
iperf3

### DISCUSSION THREAD

**Discussion by houshangaziziaccelleran** [esnet/iperf#1571] [Q&A]
Hi
Does anyone has experience installing `iperf3.9` on Ubuntu 20.04? When i use apt it gives me 3.7 and I need 3.9. copying `iperf3` from `/bin/ `of other machine does not help. It gives: 
`iperf3: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by iperf3)`

**Comment by userdocs** [comment]
because you need

option 1: build it yourself https://github.com/esnet/iperf#building

option 2: use a static build https://github.com/userdocs/iperf3-static/releases/latest
